tools/broccoli/broccoli-ts2dart.ts

```diff
@@ -4,17 +4,22 @@
 import fs = require('fs');
 import fse = require('fs-extra');
 import path = require('path');
-import * as ts2dart from 'ts2dart';
 import {wrapDiffingPlugin, DiffingBroccoliPlugin, DiffResult} from './diffing-broccoli-plugin';
 
 class TSToDartTranspiler implements DiffingBroccoliPlugin {
   static includeExtensions = ['.ts'];
 
-  private transpiler: ts2dart.Transpiler;
+  private transpiler: any /*ts2dart.Transpiler*/;
```

Comment:
https://github.com/angular/angular/commit/ae275fa4e469f0b30142260f06f2a16511dcd071#diff-cb068e9a6a1837586e67d28648bf2067L4

I'm not entirely sure. This is from the angular broccoli plugin.